src/samd/sample_init.cpp

```diff
@@ -79,7 +88,15 @@ static void initTime() {
             delay(2000);
         } else {
             Serial.print("Fetched NTP epoch time is: ");
+
+#if WIO_TERMINAL
+            char buff[32];
+            sprintf(buff, "%.f", difftime(epochTime, (time_t) 0));
```

Comment:
always just use the WIO version?

Reply:
Could possibly but I went with the idea of least amount of change to other environments to not cause unintended issues.